client/jetpack-cloud/sections/partner-portal/controller.tsx

```diff
@@ -83,9 +84,14 @@ export function licensesContext( context: PageJS.Context, next: () => void ): vo
 }
 
 export function issueLicenseContext( context: PageJS.Context, next: () => void ): void {
+	const { site_id: siteId } = context.query;
+	const state = context.store.getState();
+	const sites = getSitesItems( state );
+	const selectedSite = sites.hasOwnProperty( siteId ) ? siteId : null;
```

Comment:
I recommend replacing `sites.hasOwnProperty( siteId )` with `sites[ siteId ]` for simplicity's sake.

Also, returning `null` will not conform to the interface of the `IssueLicense` component. The interface looks like:

```
interface Props {
	selectedSite?: number;
}
```

In this case, `selectedSite` is expected to be `undefined` or a number. If we wanted to include `null` as a possible value, we would have to change the type to `selectedSite?: number | null` (the accepted types would be `undefined`, `null`, and any number).